src/util/settings/theme.cc

```diff
@@ -88,28 +89,13 @@ QColor chunkBackground(int color_index) {
   }
   QColor ret =
       chunkBackgroundColors_[color_index % chunkBackgroundColors_.size()];
-  if (isDark()) {
-    ret = QColor(ret.red() ^ 0xff, ret.green() ^ 0xff, ret.blue() ^ 0xff);
-  }
 
-  return ret;
+  return colorInvertedIfDark(ret);
 }
 
 QColor byteColor(uint8_t byte) {
-  if (isDark()) {
-    byte ^= 0xFF;
-  }
-
-  int red, blue, green = 0;
-  if (byte <= 0x80) {
-    blue = 0xff;
-    red = qMin(0xff, byte * 2);
-  } else {
-    blue = qMin(0xff, (0xff - byte) * 2);
-    red = 0xff;
-  }
-
-  return colorInvertedIfDark(QColor(red, green, blue));
+  QColor color = util::settings::hexedit::colorOfBytes();
+  return color.darker(100 + (byte ^ 0xFF) * 200 / 255);
```

Comment:
It may not be obvious for a reader that `byte ^ 0xFF` preserves correct ordering of the values, IMO `0x100 - byte` is more readable.